lua/astrocommunity/pack/clojure/init.lua

```diff
@@ -39,43 +66,34 @@ return {
       },
     },
   },
+
+  -- Treesitter structural editing
+  -- package provides normal mode key mappings
+  {
+    "PaterJason/nvim-treesitter-sexp",
+    filetype = { "clojure" },
+    dependencies = {
+      "AstroNvim/astrocore",
+      opts = {
+        -- configuration & keymaps overrides:
+        -- https://github.com/PaterJason/nvim-treesitter-sexp#configuration
+      },
+    },
+  },
+
   -- Conjure plugin for Clojure REPL
   {
     "Olical/conjure",
     -- load plugin on filetypes
-    ft = { "clojure" },
+    filetype = { "clojure" },
```

Comment:
```suggestion
    ft = { "clojure" },
```

I think this is wrong

Reply:
fixed in latest commit